README.md

```diff
@@ -39,10 +45,8 @@ Three nodes
 
 This is an example of installing PostgreSQL, OMERO.server and OMERO.web on separate nodes.
 
-- `cd three-nodes; ansible-playbook playbook.yml`
-
-Alternatively, use docker-compose to run a similar configuration in docker:
```

Comment:
I assume this gets added back when the other repo is added?

Reply:
Yes. https://github.com/ome/docker-example-omero/ has been transferred to the organization. I can certainly add them as part of this PR as well.